frontend/src/pages/TeacherProfile.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <Page blueBg>
+    <div class="teacher-profile-container">
+      <div class="teacher-profile-left">
+        <StudentProfileCard
+          v-bind:studentInfo="{
+            studentName: teacherData.FullName,
+            studentContact: teacherData.PhoneNumber,
+            source: teacherData.Source,
+            dateJoined: teacherData.dateJoined,
+            nativeLanguage: teacherData.nativeLanguage.NativeLanguage,
+            notes: teacherData.Notes,
+            status: teacherData.status.Description,
+            proficiencyLevel: teacherData.EnglishProficiency,
+          }"
+          v-bind:teacherInfo="{
+            secondLanguage: teacherData.secondLanguage.NativeLanguage,
+            languageProficiency: teacherData.LanguageProficiency,
+          }"
+          v-bind:isTeacher="true"
+          v-on:englishProficiencyIsSelected="englishProficiency = $event"
+        />
+      </div>
+      <div class="teacher-profile-right">
+        <section class="student-status-section">
+          <StatusCardScreening
+            :studentID="teacherID"
+            :englishProficiency="englishProficiency"
+            v-if="teacherData.status.Description.toUpperCase() === 'SCREENING'"
+          />
+          <StatusCardUnmatched
+            :studentID="teacherID"
+            v-if="teacherData.status.Description.toUpperCase() === 'UNMATCHED'"
+          />
```

Comment:
In `StatusCardUnmatched.vue`, `studentID` is passed to `ModalDroppedOut.vue`, which calls `updateStudentStatus(studentID)`. If we pass `teacherID`, this attempts to call `updateStudentStatus(teacherID)`, which attempts to update the student status using the teacher ID. This causes an update conflict.

To test this, observe that clicking "Drop Out" on the Teacher Profile does not change the teacher status.

Reply:
Instead, we need `StatusCardUnmatched` and `ModalDroppedOut` to switch on `isTeacher` as well.

Reply:
Thanks! let me make those changes

Reply:
Thanks! We've made the changes to ModalDroppedOut.vue, as well as the status cards for Dropped Out, Matched and Unmatched.